platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/MerkleStateRoot.java

```diff
@@ -200,7 +196,7 @@ public MerkleStateRoot() {
     @Override
     public void init(
             final Platform platform,
-            final PlatformState platformState,
+            final PlatformStateAccessor platformState,
```

Comment:
How is this `platformState` related to the field with the same name (created in the constructor)?